src/pray.c

```diff
@@ -1527,6 +1527,7 @@ dosacrifice()
                 /* Moloch's high altar */
                 if (u.ualign.record > -99)
                     u.ualign.record = -99;
+                pline("An invisible choir chants in Latin, and you are bathed in darkness...");
```

Comment:
Where's the fat lady? :-D

Reply:
That one was implemented as a YAFM, which I was going to make a separate pull request to collect all of them in.